src/components/Section/Foreign/Activities/BenefitActivity/FutureBenefit.jsx

```diff
@@ -114,6 +114,8 @@ export default class FutureBenefit extends ValidationElement {
             className="begin"
             {...this.props.Begin}
             label={i18n.t('foreign.activities.benefit.future.label.begin')}
+            minDate={(Date.now || {}).date}
```

Comment:
I don't think this is going to work correctly. `Date.now` is a function and won't have a `date` property. Additionally, the output of `Date.now()` is a value in milliseconds (representing time elapsed since Jan 1 1970).

There is probably an existing helper function in the code that will construct a date object that the app understands from `new Date()`